Packs/DeCYFIR/Integrations/decyfir/decyfir.yml

```diff
@@ -0,0 +1,56 @@
+category:    "Data Enrichment & Threat Intelligence"
+commonfields:
+  id: decyfir
+  version: -1
+configuration:
+- defaultvalue: https://20.193.159.126:443
```

Comment:
Please add to secret_ignore file.

Reply:
We corrected this.  we can push next final commit.